io.github.chidiwilliams.Buzz.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.chidiwilliams.Buzz
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: run-buzz.sh
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.secrets
+  - --device=dri
+  # TODO Switch 'all' to 'input' when it is widely available
+  #- --device=input
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
```

Comment:
why is home needed?

Reply:
In the app users can import audio and video files for transcription and subtitle generation. Different users may store files that they want to transcribe in different locations, so the whole `home` was selected.

Reply:
does this importing go through filechoosers? does it support filechooser portals ?

Reply:
GUI version uses file chooser, so for that I could look into the filechooser portals, but we also have CLI version where user can pass files to the app via CLI and a "Folder watch" option. Folder would be selected via a filechooser or entered as path in the app

Reply:
Flatpak/Flathub is primarily for GUI apps.

Reply:
Buzz is primarily a GUI app. User may prefer to enter the Folder watch path as string, also Buzz will need to see all new files added to this folder without the need for user to select each of then one by one via the file chooser.

Reply:
If it is going to default to GUI and it works with filechoosers you should either remove this or restrict it to xdg-videos downloads etc.

running CLI with flatpak is a bit weird as you have to type out a long command and people doing that should be able to pass in `--filesystem` with the directories they need.

Reply:
App defaults to GUI, limited access to typical places user may want to store audio/video files